frontend/packages/console-shared/src/components/alerts/index.ts

```diff
@@ -1,2 +1,3 @@
 export { default as AlertSeverityIcon } from './AlertSeverityIcon';
 export { default as ErrorAlert } from './error';
+export { default as DismissableAlert } from './DismissableAlert';
```

Comment:
wondering if we should remove the default export here to keep our barrel files small

```suggestion
```